core/domain/exp_services.py

```diff
@@ -305,9 +396,17 @@ def _get_exploration_summaries_from_models(exp_summary_models):
 
 
 def get_exploration_summaries_matching_ids(exp_ids):
-    """Given a list of exploration ids, return a list with the corresponding
-    summary domain objects (or None if the corresponding summary does not
-    exist).
+    """Returns a list of ExplorationSummary domain objects(or None if the
+    corresponding summary does not exist) corresponsing to the given 
+    list of exploration ids.
+
+    Args:
+        exp_ids: list(str). List of exploration ids.
+
+    Returns:
+        list(ExplorationSummary|None). List of ExplorationSummary domain objects
+        corresponding to given exploration ids. If a given ExplorationSummary
```

Comment:
given exploration ids --> the given exploration ids

a given --> an (since these are not given)